sample-compose/app/build.gradle.kts

```diff
@@ -116,60 +119,49 @@ kapt {
+    implementation(libs.androidx.datastore.preferences)
 
-    implementation("com.google.dagger:hilt-android:${Versions.HILT_VERSION}")
-    implementation("androidx.hilt:hilt-navigation-compose:${Versions.HILT_NAVIGATION_COMPOSE_VERSION}")
+    implementation(libs.bundles.hilt)
+    kapt(libs.hilt.compiler)
 
-    implementation("com.jakewharton.timber:timber:${Versions.TIMBER_LOG_VERSION}")
+    implementation(libs.timber)
 
-    implementation("com.github.nimblehq:android-common-ktx:${Versions.ANDROID_COMMON_KTX_VERSION}")
+    implementation(libs.nimble.common)
 
-    implementation("org.jetbrains.kotlin:kotlin-stdlib:${Versions.KOTLIN_VERSION}")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:${Versions.KOTLINX_COROUTINES_VERSION}")
+    implementation(libs.kotlin.stdlib)
+    implementation(libs.kotlinx.coroutines.core)
 
-    kapt("com.google.dagger:hilt-compiler:${Versions.HILT_VERSION}")
-
-    debugImplementation("com.github.chuckerteam.chucker:library:${Versions.CHUCKER_VERSION}")
-    releaseImplementation("com.github.chuckerteam.chucker:library-no-op:${Versions.CHUCKER_VERSION}")
+    debugImplementation(libs.chucker)
+    releaseImplementation(libs.chucker.noOp)
 
     // Unit test
-    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.KOTLINX_COROUTINES_VERSION}")
-    testImplementation("io.kotest:kotest-assertions-core:${Versions.TEST_KOTEST_VERSION}")
-    testImplementation("junit:junit:${Versions.TEST_JUNIT_VERSION}")
-    testImplementation("io.mockk:mockk:${Versions.TEST_MOCKK_VERSION}")
-    testImplementation("app.cash.turbine:turbine:${Versions.TEST_TURBINE_VERSION}")
+    testImplementation(libs.bundles.unitTest)
+    testImplementation(libs.test.turbine)
 
     // UI test with Robolectric
-    testImplementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
-    testImplementation("androidx.compose.ui:ui-test-junit4")
-    testImplementation("androidx.test:rules:${Versions.TEST_RULES_VERSION}")
-    testImplementation("org.robolectric:robolectric:${Versions.TEST_ROBOLECTRIC_VERSION}")
+    testImplementation(platform(libs.compose.bom))
+    testImplementation(libs.test.compose.ui)
+    testImplementation(libs.test.rules)
+    testImplementation(libs.test.robolectric)
 
     // UI test
-    androidTestImplementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
-    androidTestImplementation("androidx.compose.ui:ui-test-junit4")
-    androidTestImplementation("androidx.test:rules:${Versions.TEST_RULES_VERSION}")
-    androidTestImplementation("io.mockk:mockk-android:${Versions.TEST_MOCKK_VERSION}")
+    androidTestImplementation(platform(libs.compose.bom))
+    androidTestImplementation(libs.test.compose.ui)
+    androidTestImplementation(libs.test.rules)
+    androidTestImplementation(libs.test.mockk)
 }
 
 /*
```

Comment:
Please migrate the kover config
```
/*
 * Kover configs
 */
dependencies {
    kover(project(":data"))
    kover(project(":domain"))
}
```

Reply:
Would it be possible referencing from version catalogs?

```
/*
 * Kover configs
 */
dependencies {
    kover(projects.data)
    kover(projects.domain)
}
```